### PR TITLE
Selection overlay

### PR DESCRIPTION
**Pull request by antonvas**
Added overlay to selection for https://github.com/partnermarketing/pm-image-editor/issues/8

Sample: http://partnermarketing.github.io/pm-image-editor/index.html?branch=selection-overlay

### CODE REVIEW

File: src/scss/pm-image-editor.scss [hunk 1]
```diff
@@ -14,16 +14,52 @@ image-editor {
...
+    .image-editor-selection-border-bottom {
+        bottom: 0
+    }
+    .image-editor-selection-border-left,
+    .image-editor-selection-border-right {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 1px;
+        background-image: url('data:image/gif;base64,R0lGODlhAQAGAKEAAP///wAAADY2Nv///yH/C05FVFNDQVBFMi4wAwEAAAAh/hpDcmVhdGVkIHdpdGggYWpheGxvYWQuaW5mbwAh+QQACgD/ACwAAAAAAQAGAAACAxQuUgAh+QQBCgADACwAAAAAAQAGAAACA5SAUgAh+QQBCgADACwAAAAAAQAGAAACA5SBBQAh+QQBCgADACwAAAAAAQAGAAACA4QOUAAh+QQBCgADACwAAAAAAQAGAAACAwSEUAAh+QQBCgADACwAAAAAAQAGAAACA4SFBQA7');
```
Comment (amyboyd) [inline src/scss/pm-image-editor.scss:48]: Can you comment where the source image file is for this? So if we need to update the icon, we know where to look.

File: src/scss/pm-image-editor.scss [hunk 2]
```diff
@@ -14,16 +14,52 @@ image-editor {
...
       cursor: move;
     }
+
+    .image-editor-selection-border-top,
+    .image-editor-selection-border-bottom {
+        position: absolute;
+        left: 0;
+        right: 0;
+        height: 1px;
+        background-image: url('data:image/gif;base64,R0lGODlhBgABAKEAAP///wAAADY2Nv///yH/C05FVFNDQVBFMi4wAwEAAAAh/hpDcmVhdGVkIHdpdGggYWpheGxvYWQuaW5mbwAh+QQACgD/ACwAAAAABgABAAACAxQuUgAh+QQBCgADACwAAAAABgABAAACA5SAUgAh+QQBCgADACwAAAAABgABAAACA5SBBQAh+QQBCgADACwAAAAABgABAAACA4QOUAAh+QQBCgADACwAAAAABgABAAACAwSEUAAh+QQBCgADACwAAAAABgABAAACA4SFBQA7');
```
Comment (amyboyd) [inline src/scss/pm-image-editor.scss:32]: Can you comment where the source image file is for this? So if we need to update the icon, we know where to look.

**Comment by antonvas** [comment]
Done @amyboyd

**Comment by amyboyd** [comment]
+1 - having the compiled files in Git is still wrong though - you can see how it messes up the commits and diffs in here.

**Comment by antonvas** [comment]
Yes. But now I can do the trick like: http://partnermarketing.github.io/pm-image-editor/index.html?branch=selection-overlay and shouldnt update gh-pahe every time when new feature came up

**Comment by osavchenko** [comment]
:+1:

**Pull request merged**